src/redux/reducers/loginReducer.js

```diff
@@ -0,0 +1,35 @@
+import {
+  LOGIN_SUCCESSFUL,
+  LOGIN_ERROR,
+  LOGOUT_SUCCESSFUL
+} from "../action_types";
+
+export const initialState = {
+  message: {},
+  error: {},
+  isAuthenticated: false
+};
```

Comment:
You don't seem to have an `initialState` file. No need to create one as it has already been created by @micahoriaso . So when you rebase, you don't have to define `initialState` as you have right here. All you have to do is import `initialState` then create an object within `initialState` file to hold the contents from line 8-10 then on line 12 access `state = initialState.objectYouHaveCreated`.